relay-general/src/store/normalize/breakdowns.rs

```diff
@@ -31,7 +31,7 @@ impl TimeWindowSpan {
         }
     }
 
-    fn get_duration(&self) -> f64 {
+    pub fn get_duration(&self) -> f64 {
```

Comment:
_nit_: Usually, there are no `get_` prefixes.

```suggestion
    pub fn duration(&self) -> f64 {
```